packages/b/buckets.json

```
@@ -24,13 +24,13 @@
     "map"
   ],
   "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/mauriciosantos/Buckets-JS.git",
+    "source": "npm",
+    "target": "buckets-js",
```

```suggestion
    "source": "git",
    "target": "git://github.com/mauriciosantos/Buckets-JS.git",
```
The [git repo has the files needed and tagged correctly](https://github.com/mauriciosantos/Buckets-JS/tree/v1.98.2/dist), so let's avoid complicating the change and stick with git as the source for files.